Assets/Scripts/FloatingOrigin/WorldTransform.cs

```diff
@@ -26,9 +30,6 @@ private void Awake()
             {
                 worldTransformShifter = gameObject.AddComponent<GameObjectWorldTransformShifter>();
             }
-
-            // Pre-initialize the coordinates before using them
-            Coordinate = new Coordinate(ReferenceCoordinateSystem, 0, 0, 0);
```

Comment:
Kan je deze terugplaatsen? Want in de PrepareToShift zal na deze change een null value doorgegeven worden; door dit in te stellen krijg je type safety en minder null exception

Reply:
done